docs/development.md

```diff
@@ -57,6 +57,24 @@ cd foreman
 bundle exec foreman start
 ```
 
+### Stable Katello Development Box
+
+There is a stable box image of a full Katello development install published to Vagrant cloud nightly. In the `vagrant/boxes.d/99-local.example` file, a box using this image is available under the name `centos7-katello-devel-stable`
+
+```
+vagrant up centos7-katello-devel-stable
+```
+
+Once you have spun up the box, to create a new `centos-katello-devel-stable` box using the latest image from vagrant cloud you will need to update the underlying box image. For example:
+```
+vagrant destroy centos7-katello-devel-stable
+vagrant box update centos7-katello-devel-stable
```

Comment:
I can't recall exactly. Does the libvirt plugin handle when `box update` is run properly? Or does an additional command have to be run?

Reply:
that is a good point, I updated the docs with the info to remove the old box definitions from both vagrant and libvirt, hopefully things are more clear now